pkg/bundle/bundle.go

```diff
@@ -202,7 +202,26 @@ func (b *bundle) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result,
 			return ctrl.Result{Requeue: true}, b.targetDirectClient.Status().Update(ctx, &bundle)
 		}
 
-		if synced {
+        secretSynced := false
+        if bundle.Spec.Target.Secret != nil {
+            var syncErr error
+            secretSynced, syncErr = b.syncSecretTarget(ctx, log, &bundle, namespaceSelector, &namespace, []byte(resolvedBundle.data))
+            if syncErr != nil {
+                log.Error(syncErr, "failed sync bundle to target namespace")
+                b.recorder.Eventf(&bundle, corev1.EventTypeWarning, "SyncTargetFailed", "Failed to sync secret target in Namespace %q: %s", namespace.Name, syncErr)
+
+                b.setBundleCondition(&bundle, trustapi.BundleCondition{
+                    Type:    trustapi.BundleConditionSynced,
+                    Status:  corev1.ConditionFalse,
+                    Reason:  "SyncTargetFailed",
+                    Message: fmt.Sprintf("Failed to sync bundle target secret to namespace %q: %s", namespace.Name, syncErr),
+                })
+
+                return ctrl.Result{Requeue: true}, b.targetDirectClient.Status().Update(ctx, &bundle)
+            }
+        }
+
+		if synced || secretSynced {
```

Comment:
suggestion: Would it be a good idea here to rename `synced` to `configMapSynced` so it's clearer what that variable is doing?

The log messages and error messages for the config map sync above could maybe also use updating to directly mention that they're for configmaps

Reply:
I agree. Did the necessary changes.